index.bs

```diff
@@ -4152,6 +4124,11 @@ Constructors</h4>
 <dl dfn-type=constructor dfn-for="AnalyserNode">
 	: <dfn>AnalyserNode(context, options)</dfn>
 	::
+		When the constructor is called with a {{BaseAudioContext}} <var>c</var> and
+		an option object <var>option</var>, the user agent MUST <a
+		href="#audionode-constructor-init">initialize the AudioNode</a>
```

Comment:
Isn't there a bikeshed way to reference a section of the doc instead of using href?

Reply:
Non, only sections for some reason.